packages/eventual-send/src/index.js

```diff
@@ -361,27 +361,26 @@ export function makeHandledPromise() {
       if (method === undefined || method === null) {
         if (!(t instanceof Function)) {
           const ftype = typeof t;
-          throw TypeError(`target is not a function, typeof is ${ftype}`);
+          throw TypeError(
+            `method name was undefined. (target's type is "${ftype}")`,
```

Comment:
I tested this manually in the debugger, by changing a message to `undefined` just before this point. If you can think of a way to add a test that sends `undefined`, I'd add it to the tests.

Reply:
> If you can think of a way to add a test that sends `undefined`, I'd add it to the tests.

`E(target)(...args)` sends `undefined`, as it is a function invocation, not a method call (though not supported by our current marshal layer).

You should be able to get it to work/fail somewhere in the eventual-send tests.